samples/subapp1-online-store/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "subapp1-online-store",
+  "version": "0.2.1",
+  "description": "SubApp Redux POC",
+  "homepage": "http://test",
+  "author": {
+    "name": "Mateja Ristic",
+    "email": "",
+    "url": "https://github.com/mristic505"
+  },
+  "contributors": [],
+  "files": [
+    "server",
+    "src",
+    "lib",
+    "dist"
+  ],
+  "main": "lib/server/index.js",
+  "keywords": [
+    "test",
+    "electrode"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "test/test-app"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">= 8",
+    "npm": ">= 5"
+  },
+  "scripts": {
+    "dev": "clap -q dev",
+    "test": "clap check",
+    "build": "clap build"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.17.9",
+    "@module-federation/concat-runtime": "^0.0.1",
+    "@xarc/fastify-server": "^3.3.0",
+    "electrode-confippet": "^1.7.1",
+    "history": "^5.3.0",
+    "html-webpack-plugin": "^5.5.0",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-redux": "^8.0.1",
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
+    "redux": "^4.2.0",
+    "redux-logger": "^3.0.6"
+  },
+  "devDependencies": {
+    "@xarc/app-dev": "^10.1.1",
+    "@xarc/opt-eslint": "^2.0.1"
+  },
+  "fyn": {
+    "dependencies": {
+      "@xarc/app": "../../packages/xarc-app",
+      "subapp-react": "../../packages/subapp-react",
+      "subapp-redux": "../../packages/subapp-redux",
+      "subapp-server": "../../packages/subapp-server",
+      "subapp-web": "../../packages/subapp-web"
+    },
+    "devDependencies": {
+      "@xarc/app-dev": "../../packages/xarc-app-dev",
+      "@xarc/opt-eslint": "../../packages/xarc-opt-eslint"
+    }
```

Comment:
nit: Does `fun bootstrap` do this for us?

Reply:
Many samples use this approach by adding packages manually to FYN object. I am investigating options for linking dependencies with FYN.

Reply:
Thanks! I've seen that this fyn deps object is the common approach. Normally monorepo managers manage dependency linking without having to re-declare dependencies, so please share if you see that there's a better way with fyn.